app/src/main/res/layout/activity_bmi_calculator.xml

```diff
@@ -163,7 +168,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:gravity="center_horizontal"
-                android:text="Your BMI"
+                android:text="@string/your_bmi"
```

Comment:
Fixed the error, redesigned the page layout and added the missing strings.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk